gsadjust/pyqt_models.py

```diff
@@ -136,28 +143,24 @@ def grav(self):
         data = np.array(self.raw_grav) - np.array(self.tare) + np.array(self.etc)
         return data.tolist()
 
-    @property
```

Comment:
@mfitzp Does deleting the @property tags require adding parenthesis to all usages in other files? I.e, changing ObsTreeItem.name to ObsTreeItem.name() ?

Reply:
@jkennedy-usgs yes it will. In cases where that isn't ideal you could also use getattr to turn the property access into a function call, e.g.

COLUMN: (getattr, self, 'name_of_attribute')

Reply:
Ah, that makes sense. I didn't put 2 and 2 together to realize the conversion from property to function is required to make the "return a function" work.

Reply:
It's not always obvious with @property as you've got what looks very much like a function, but it isn't. The goal with all these changes is just to postpone the point at which we evaluate the function (run what's in it). With a @property any time you access it, the function is run.

You can also postpone it using a lambda COLUMN: (lambda: self.name_of_attribute, ) if you like. If you do this in the method you would create a new lambda *every* time (which is worse) ....but if you define your dict on the class it's a one-off. You could also do something with `partial`.

There are a lot of ways to do this sort of thing in Python :)